backend/functions/lib/adapters/services/logger.service.ts

```diff
@@ -9,14 +9,19 @@ const disallowedProperties = ['ssn', 'taxId'];
 
 export class LoggerImpl implements LoggerHelper {
   private readonly provider: LoggerProvider;
-  constructor(provider: LoggerProvider = console.log) {
+  private readonly invocationId: string;
+
+  constructor(invocationId: string, provider: LoggerProvider = console.log) {
     this.provider = provider;
+    this.invocationId = invocationId;
```

Comment:
Now that we've added invocation id, we should consider adding something like a correlation id in the future. Doesn't need to happen now, but something to think about as we move forward for tracking related requests by the client.

Reply:
💯